app/controllers/dates_controller.rb

```diff
@@ -196,7 +196,8 @@ def confirm_weekly_charge
   #
   def select_weekly_date
     @local_authority = la_id
-    @dates = Dates::Weekly.call(vrn: vrn, zone_id: @local_authority)
+    @charge_start_date = FetchSingleCazData.call(zone_id: la_id)&.active_charge_start_date
```

Comment:
I think that we should use `@local_authority` here, instead of `la_id` directly, because also in the one line below we can use that.
```suggestion
    @charge_start_date = FetchSingleCazData.call(zone_id: @local_authority)&.active_charge_start_date
```